aiida_common_workflows/workflows/relax/siesta/workchain.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+WorkChain that runs SiestaBaseWorkChain of aiida_siesta package and
+returns the agreed outputs.
+"""
+from aiida.orm import (Float, ArrayData)
+from aiida.engine import calcfunction
+from aiida_siesta.workflows.base import SiestaBaseWorkChain
```

Comment:
would be best to use the `WorkflowFactory`

Reply:
All good, but I don't think you have addressed this one. If you change it, I will approve

Reply:
I did, why isn't it recognised?

Reply:
Ok, nvm, I see the change now, sorry